release-tools/prow.sh

```diff
@@ -890,7 +890,6 @@ EOF
 #!/bin/sh
 if ! kubectl exec "${CSI_PROW_SANITY_POD}" -c "${CSI_PROW_SANITY_CONTAINER}" -- rmdir "\$@"; then
     kubectl exec "${CSI_PROW_SANITY_POD}" -c "${CSI_PROW_SANITY_CONTAINER}" -- rm -rf "\$@"
-    exit 1
```

Comment:
I have no idea about the reason for exiting. But I think we should not exit the script here because we do need to delete the directory. @pohly 

After removing it, all tests pass. Currently, the failure reason is that I should change it in `csi-release-tools` repo.

Reply:
The `exit 1` is called if a simple `rmdir` has failed, typically because the driver hasn't remove its content from that directory. Why do you think that it needs to be removed? All that it does is cause the script and thus the test to fail; the cleanup still happens.

As you said, if this is a valid change, then it needs to go into `csi-release-tools` first.

Reply:
In my opinion, whether `rmdir`  succussed or not, the directory will be removed in the original script. So the exit will not make sense.